### PR TITLE
ref(app-store-connect): Fixed verification msg code alignment & add notifications

### PR DESCRIPTION
**Pull request by priscilawebdev**
. displays notification when a SMS or an iTunes verification code has been sent 
. Fixed alert alignment:

**Before:**
image

image

**After:**

image

image -> here I would like to have the button centralized, but I will do it maybe in a follow-up PR

### CODE REVIEW

**Review by dashed — APPROVED** [review]

File: static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx [hunk 1]
```diff
@@ -320,13 +320,16 @@ function AppStoreConnect({
...
       }
+
+      addSuccessMessage(t('An iTunes verification code has been sent'));
     } catch {
       if (shouldGoNext) {
         setIsLoading(false);
       }
       addErrorMessage(
-        t('The iTunes authentication failed. Please check the entered credentials.')
+        t('The iTunes authentication failed. Please check the entered credentials')
```
Comment (dashed) [inline static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx:332]: This phrasing seems weird:

> Please check the entered credentials

Maybe

> Please check the given credentials

Comment (markstory) [reply static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx:332]: Or 'Please check the provided credentials'

File: static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepFour.tsx [hunk 2]
```diff
@@ -78,13 +78,22 @@ export default StepFour;
 
 const StyledAlert = styled(Alert)`
   div {
-    align-items: center;
+    align-items: flex-start;
+  }
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
```
Comment (markstory) [inline static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepFour.tsx:83]: Should we consider promoting this layout to all Alerts that also contain buttons?

**Review by markstory — APPROVED** [review]

File: static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx:332]: I will update the code..thanks for suggestions!

File: static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepFour.tsx (hunk 2 above)
Comment (priscilawebdev) [reply static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepFour.tsx:83]: yes. thought the same thing.. I will do this change in a follow-up PR 😉

**Pull request merged**